docs/developer-docs/integrations/sns/tokenomics/sns-checklist.md

```diff
@@ -139,7 +122,7 @@ It’s strongly recommended that developers have a discussion in the forum with
 
 It is recommended to share:
 
-- [x] The init file and swap file (example: [OpenChat](https://github.com/open-ic/open-chat/tree/master/sns/config)). <!-- NEW: init file-->
+- [x] The init file and swap file (example: [OpenChat](https://github.com/open-ic/open-chat/tree/master/sns/config)). <!-- NEW: TODO init file-->
```

Comment:
@DanielThurau do we happen to have a new example file here? If not, I can just remove this line..

Reply:
Also, @DanielThurau, we don't have a swap file, right? I would remove this to just say "the init file" (used to say "the init file and swap file"). please let me know if this is wrong

Reply:
Confirmed we do not have a swap file and you should delete it. I am currently working on a template that we can include in the docs of the new configuration file. I will have that ready for you in the next couple days.

Reply:
Here is the link https://github.com/dfinity/ic/blob/master/rs/sns/cli/sns_init_template.yaml

Reply:
Linked to it, thanks!